### PR TITLE
Add a note about buttons accessibility to docs

### PR DESCRIPTION
**Pull request by adamczyk777**
Just a note on how to make buttons accessible for now.

Related to issue #468

### CODE REVIEW

File: docs/component-buttons.md [hunk 1]
```diff
@@ -16,6 +16,24 @@ Currently Gesture handler library exposes three components that render native to
...
+    </RectButton>);
+  // Accessible:
+  const AccessibleButton = () => (
+    <RectButton onPress={this._onPress}>
+      <View accessible>
+        <Text>Bar</Text>
+      </View>
+    </RectButton>);
+```
+If your children are already wrapped in some kind of `View` just add `accessible` prop to that View.
```
Comment (osdnk) [inline docs/component-buttons.md]: I think it's not needed

**Comment by satya164** [comment]
Maybe this should be built-in? The button is a common component, so accessibility should be baked in. Also, many people don't bother to add accessibility related props.

**Comment by osdnk** [comment]
@satya164 
Maybe, but I wanna omit wrapping and cloning.

**Comment by adamczyk777** [comment]
Updated the docs notice. It affects both iOS and Android.

**Pull request merged**